xtask/src/templates/control_id_zkr.rs

```diff
@@ -16,3 +16,5 @@
 pub const ALLOWED_IDS_ROOT: &str = "{}";
 
 pub const RECURSION_CONTROL_IDS: [(&str, &str); {}] = [{}];
+
+pub const RECURSION_PROGRAM_ID: &str = "{}";
```

Comment:
I think maybe this didn't get updated?

Reply:
Thank you! Failed to save the file. Should be fixed now: I've re-run `cargo xtask bootstrap` (which is what this would break) and it changes no files.